modules/proj4/src/lib/proj4-projection.js

```diff
@@ -12,20 +12,12 @@ export class Proj4Projection {
   }
 
   constructor({from = 'WGS84', to = 'WGS84'}) {
-    this.projection = proj4(from, to);
-    if (!this.projection) {
+    const projection = proj4(from, to);
+    if (!projection) {
       throw new Error('Invalid projection');
     }
 
-    this.project = this.project.bind(this);
-    this.unproject = this.unproject.bind(this);
-  }
-
-  project(coords) {
-    return this.projection.forward(coords);
-  }
-
-  unproject(coord) {
-    return this.projection.inverse(coord);
+    this.project = projection.forward.bind(this);
```

Comment:
This seems strange as `forward` knows nothing about `Proj4Projection`, so why would it benefit from being bound.

This could perhaps make sense `this.project = projection.forward.bind(projection);` though it is not clear that this would have an effect given how the projection object is implemented.

Does `this.project = projection.forward` work as well?

While this saves the overhead of one call, wiring up methods like this feels a bit hacky to me.

Reply:
> Does `this.project = projection.forward` work as well?

Yes you're right, it does work without the `bind(this)`.

> wiring up methods like this feels a bit hacky to me

I can revert to the existing code and search more for how to correctly bind the class' methods, but given that we aren't changing the function signature of `forward` at all, it seems ok to me.